.github/workflows/release-integration.yml

```diff
@@ -95,10 +99,79 @@ jobs:
             -t $DOCKER_IMAGE_NAME:$DOCKER_IMAGE_TAG \
             -t $DOCKER_IMAGE_NAME:latest \
             .
+  open-pr:
+    name: Update version and appVersion and open pr
+    needs: [ docker-integration ]
+    runs-on: ubuntu-latest
+    # run only for releases (not prereleases)
+    if: ${{ ! github.event.release.prerelease }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Find new appVersion
+        id: find-version
+        run: |
+          echo "NEW_APP_VERSION=${{ needs.docker-integration.outputs.new-version }}" >> $GITHUB_ENV
+          echo "new app version: $NEW_APP_VERSION"
+
+      - name: Find current appVersion
+        id: original_version
+        run: |
+          ORIGINAL_APP_VERSION=$(yq eval '.appVersion' ${{ env.CHART_DIRECTORY }}/Chart.yaml)
+          echo "original app version: $ORIGINAL_APP_VERSION"
+          echo "ORIGINAL_APP_VERSION=$ORIGINAL_APP_VERSION" >> $GITHUB_ENV
+
+      - name: Find current helm chart version
+        run: |
+          CURRENT_VERSION=$(yq eval '.version' ${{ env.CHART_DIRECTORY }}/Chart.yaml)
+          echo "version: $CURRENT_VERSION"
+          echo "CURRENT_VERSION=$CURRENT_VERSION" >> $GITHUB_ENV
+
+      - name: Run version update custom action
+        id: run-reusable
+        uses: ./.github/actions/version-bump
+        with:
+          currentVersion: $CURRENT_VERSION
+          currentAppVersion: $ORIGINAL_APP_VERSION
+          updatedAppVersion: $NEW_APP_VERSION
+
+      - name: Update version helm chart
+        # fail the workflow if newVersion is "error", otherwise set the new versions and continue with opening pr
+        run: |
+          if [ "${{ steps.run-reusable.outputs.newVersion }}" != 'error' ]; then
+            echo "new appVersion to set: ${NEW_APP_VERSION}"
+            echo "new version to set: ${{ steps.run-reusable.outputs.newVersion }}"
+            yq e -i ".appVersion=\"${NEW_APP_VERSION}\"" "${{ env.CHART_DIRECTORY }}/Chart.yaml"
+            yq e -i '.version="${{ steps.run-reusable.outputs.newVersion }}"' ${{ env.CHART_DIRECTORY }}/Chart.yaml
+          else
+            echo "Error: newVersion is 'error'."
+            exit 1
+          fi
+
+      - name: Configure Git
+        run: |
+          git config user.name "${{ github.actor }}"
+          git config user.email "${{ github.actor }}@users.noreply.github.com"
+      
+      - name: Commit Changes
+        run: |
+          git checkout -b update-chart-version-${{ github.sha }}
+          git branch -a
+          git add ${{ env.CHART_DIRECTORY }}/Chart.yaml
+          git commit -m "Bump version and appVersion"
```

Comment:
We also need to run `helm-docs` to update the chart's `README.md`. If it helps, we can even create a single PR for bumping versions and updating the README file